mathics_django/web/format.py

```diff
@@ -43,8 +46,6 @@ def eval_boxes(result, fn, obj, **options):
     if isinstance(format, dict):
         return dict((k, obj.format_output(expr, f)) for k, f in format.items())
 
-    from mathics.core.expression import Expression, BoxError
```

Comment:
Why remove it from here? Isn't lazy reload good?

Reply:
You spend _way_ too much time worrying about this kind of stuff

Expression is needed to format just about everything. 

But tell you what - run some tests to see how much time you save doing this. I doubt you will find a difference. But prove me wrong!

Reply:
> You spend way too much time worrying about this kind of stuff

Ok, I'll try to stop doing this.